### PR TITLE
Add a method to create a new PBXVariantGroup in a group

### PR DESCRIPTION
**Pull request by timbodeit**
Used for fixing https://github.com/CocoaPods/CocoaPods/issues/2597

Please provide feedback on the test coverage.

### CODE REVIEW

**Comment by segiddins** [comment]
Looks good to me

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,13 @@
 # Xcodeproj Changelog
 
+## Master
+
+##### Enhancements
+
+* Add method to create new variant groups (groups for localized versions of the same file)  
+  [Tim Bodeit](https://github.com/timbodeit)
+  [Xcodeproj#301](https://github.com/CocoaPods/Xcodeproj/pull/315)
```
Comment (segiddins) [inline CHANGELOG.md]: is this supposed to be 301 or 315? Also, needs an extra newline here

**Comment by segiddins** [comment]
:+1: other than commend

File: CHANGELOG.md (hunk 1 above)
Comment (timbodeit) [inline CHANGELOG.md]: Very good catch! I think I copied the line to match the rest of the changelog and only changed one of the PR numbers.

**Comment by segiddins** [comment]
:shipit:

**Pull request merged**